mealie/services/migrations/_migration_base.py

```diff
@@ -109,6 +111,7 @@ def _save_all_entries(self) -> None:
         if not is_success and not is_failure:
             self.report.status = ReportSummaryStatus.partial
 
+        self.report.entries = new_entries
```

Comment:
I don't know why this used to work. Since we're saving the report again without updating the report's entry list, it should delete all the report entries that we just created. It's doing that _now_, but I don't know why it wasn't doing that before